star_align.py

```diff
@@ -70,6 +70,48 @@ def scan_workdir(base):
     raise Exception("Unable to determine input type")
 
 
+def spreadsheet2dict(spreadFile):
```

Comment:
My use case is a little different than the standard library. Whereas DictReader provides an iterator over rows, where I can address each record with the field names defined in the header row, my function returns a dictionary that is accessible by analysis ID. This way I can easily look up the record I need. Anyway, I will check if it is easier to use the reader to construct my dictionary.